services/marko-example-website/server/templates/magazine/components/layout.marko

```diff
@@ -0,0 +1,22 @@
+import { getAsObject } from '@base-cms/object-path';
+
+$ const publications = getAsObject(input, 'publications');
+
+<document>
+  <@head>
+    <cms-page-metadata for="magazine" ...publications />
```

Comment:
Make a distinct layout template per type. Update `cms-page-metadata for "magazine-publication", etc. Also, manually add title and meta description for the root publication index using `cms-page-title` and `cms-page-meta-description`